app/src/org/commcare/dalvik/activities/UnrecoverableErrorActivity.java

```diff
@@ -43,7 +39,7 @@ protected Dialog onCreateDialog(int id) {
         AlertDialogFactory factory = new AlertDialogFactory(this, title, message);
         DialogInterface.OnClickListener buttonListener = new DialogInterface.OnClickListener() {
             public void onClick(DialogInterface dialog, int i) {
-               Intent intent = new Intent(UnrecoverableErrorActivity.this, CommCareHomeActivity.class);
+               Intent intent = new Intent(UnrecoverableErrorActivity.this, DispatchActivity.class);

```

Comment:
Pretty strange that we restart the app when it reaches an unrecoverable error. Especially because the dialog says "Shutdown CommCare".
This most likely creates a loop for the user, because when the app restarts I assume it will reach the same error. 
Anyone know why this is here?